### ISSUE TITLE
[Bug]: cannot cp to host storage from archlinux

### ISSUE THREAD

**Issue by champignoom** [termux/proot-distro#289]
### Problem description

Cannot cp file from archlinux to mounted host storage: permission denied.

But cat \<fromfile \>tofile succeeds and does the same job.

### What steps will reproduce the bug?

```
~ $ termux-setup-storage
~ $ proot-distro login archlinux --bind $HOME/storage:/mnt/storage
[root@localhost ~]# echo ppp >ppp.txt
[root@localhost ~]# cp ppp.txt /mnt/storage/downloads
cp: failed to clone '/mnt/storage/downloads/ppp.txt' from 'ppp.txt': Permission denied
[root@localhost ~]# cat <ppp.txt >/mnt/storage/downloads/ppp.txt
[root@localhost ~]# cat /mnt/storage/downloads/ppp.txt
ppp
```

**Comment by TomJo2000** [comment]
Most likely related to [this issue over on termux-packages](https://github.com/termux/termux-packages/issues/15706), regarding coreutils.

**Comment by champignoom** [comment]
> Most likely related to [this issue over on termux-packages](https://github.com/termux/termux-packages/issues/15706), regarding coreutils.

Indeed. Closing as this will be fixed by the next bugfix release of coreutils 9.2 .

**Comment by champignoom** [comment]
Reopening as the bug persists in [coreutils 9.2-2](https://archlinuxarm.org/packages/aarch64/coreutils) released on Mar 26.

**Comment by phyr1sc** [comment]
On archlinux:
```
cp: failed to clone '/storage/emulated/0/Download/pp.txt' from 'pp.txt': Inappropriate ioctl for device
```

On the termux shell (on coreutils 9.2-4)
![Screenshot_2023-03-28-18-36-54-71](https://user-images.githubusercontent.com/83752061/228210588-58d0b8b0-a354-4db7-bdaa-7d2db691db6a.png)
![Screenshot_2023-03-28-18-38-58-63](https://user-images.githubusercontent.com/83752061/228210722-2c5ab6d3-82ef-4ad1-8dbf-f683b10b9c5b.png)

**Comment by phyr1sc** [comment]
However it works fine using an external command like the Midnight Commander `mc`.

**Comment by champignoom** [comment]
> On archlinux:
> 
> ```
> cp: failed to clone '/storage/emulated/0/Download/pp.txt' from 'pp.txt': Inappropriate ioctl for device
> ```
> 
> On the termux shell (on coreutils 9.2-4) ![Screenshot_2023-03-28-18-36-54-71](https://user-images.githubusercontent.com/83752061/228210588-58d0b8b0-a354-4db7-bdaa-7d2db691db6a.png) ![Screenshot_2023-03-28-18-38-58-63](https://user-images.githubusercontent.com/83752061/228210722-2c5ab6d3-82ef-4ad1-8dbf-f683b10b9c5b.png)

It looks like it's not a problem of host-guest interaction but rather a problem of coreutils/cp whose bugfix has either not been finished or not been adapted by archlinuxarm, as the current archlinux cannot even cp files within itself.

**Comment by phyr1sc** [comment]
> However it works fine using an external command like the Midnight Commander `mc`.

I stated a workaround; it always works

**Comment by phyr1sc** [comment]
the ioctl errors doesn't appear on manjaro, which is weird since it's based on arch
anyways, use manjaro if you want an archlinux experience
